ams/server/src/main/java/com/netease/arctic/server/dashboard/model/TagOrBranchInfo.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netease.arctic.server.dashboard.model;
+
+import org.apache.iceberg.SnapshotRef;
+
+public class TagOrBranchInfo {
+  public static final TagOrBranchInfo MAIN_BRANCH =
+      new TagOrBranchInfo(SnapshotRef.MAIN_BRANCH, -1, -1, 0L, 0L, "branch");
+
+  private String name;
+  private long snapshotId;
+  private Integer minSnapshotsToKeep;
+  private Long maxSnapshotAgeMs;
+  private Long maxRefAgeMs;
+  private String type;
+
+  public TagOrBranchInfo() {}
+
+  private TagOrBranchInfo(
+      String name,
+      long snapshotId,
+      Integer minSnapshotsToKeep,
+      Long maxSnapshotAgeMs,
+      Long maxRefAgeMs,
+      String type) {
+    this.name = name;
+    this.snapshotId = snapshotId;
+    this.minSnapshotsToKeep = minSnapshotsToKeep;
+    this.maxSnapshotAgeMs = maxSnapshotAgeMs;
+    this.maxRefAgeMs = maxRefAgeMs;
+    this.type = type;
+  }
+
+  public TagOrBranchInfo(String name, SnapshotRef snapshotRef) {
+    this.name = name;
+    this.snapshotId = snapshotRef.snapshotId();
+    this.minSnapshotsToKeep = snapshotRef.minSnapshotsToKeep();
+    this.maxSnapshotAgeMs = snapshotRef.maxSnapshotAgeMs();
+    this.maxRefAgeMs = snapshotRef.maxRefAgeMs();
+    if (snapshotRef.isTag()) {
+      this.type = "tag";
```

Comment:
We should use constants(`public static final String xxxx`) to replace the using of "tag" and "branch".

Reply:
done.